easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,133 @@ def extra_options():
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
         }
-        return ConfigureMake.extra_options(extra_vars)
+        extra_vars.update(ConfigureMake.extra_options())
```

Comment:
Doesn't CMakeMake already do this?

Reply:
mmmm, I don't know. I copied that pattern from the bamtools EasyBlock

Reply:
The `CMakeMake` block indeed already does this, so just doing `return CMakeMake.extra_options(extra_vars)` is enough.